introspection-engine/connectors/sql-introspection-connector/tests/db_specific_introspection/mysql/tables_mysql.rs

```diff
@@ -340,3 +340,26 @@ async fn introspecting_a_table_enums_should_return_alphabetically_even_when_in_d
     custom_assert(&result3, dm);
     custom_assert(&result4, dm);
 }
+
+#[test_each_connector(tags("mysql"))]
+async fn introspecting_a_table_without_uniques_should_comment_it_out(api: &TestApi) {
+    api.barrel()
+        .execute(|migration| {
+            migration.create_table("User", |t| {
+                t.add_column("id", types::primary());
+            });
+            migration.create_table("Post", |t| {
+                t.add_column("id", types::integer());
+                t.inject_custom(
+                    "user_id INTEGER NOT NULL UNIQUE,
+                FOREIGN KEY (`user_id`) REFERENCES `User`(`id`)",
+                )
+            });
+        })
+        .await;
+
+    let dm = "/// The underlying table does not contain a unique identifier and can therefore currently not be handled.\n// model Post {\n  // id      Int\n  // user_id User\n// }\n\nmodel User {\n  id Int @default(autoincrement()) @id\n}";
```

Comment:
Are these always rendered on a single line? It's not the nicest for users, I wonder how hard it would be to change that.

Reply:
No they are not, it was just the easiest way to get the format to be the same in the test.